src/assets/styles/fonts.scss

```
@@ -78,9 +78,98 @@
+  font-display: swap;
+  font-family: HelsinkiGroteskPro;
+  font-style: italic;
+  font-weight: 900;
+  src: url('https://makasiini.hel.fi/delivery/917fc4ffc37c44d31b143244042e0078/9a677fe2c4414b1a40990971cd4ac24c.woff2')
+    format('woff2');
+  text-rendering: optimizeLegibility;
+}
+
+$hel-grotesk:
+  'HelsinkiGrotesk',
+  Arial,
```

Stylelint (17.12.0)

[error] Expected "Arial" to be "arial" (value-keyword-case)

(value-keyword-case)